### ISSUE TITLE
Exception handling broken?

### ISSUE THREAD

**Issue by uliska** [frescobaldi/frescobaldi#1167]
I fear I've broken the exception handling. I notice that recently I didn't get the exception dialog anymore but terminal exceptions like this:

```
Traceback (most recent call last):
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/__init__.py", line 67, in showAvailableFonts
    dialog.show_fonts_dialog(self.mainwindow())
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/dialog.py", line 58, in show_fonts_dialog
    dlg = FontsDialog(mainwin)
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/dialog.py", line 92, in __init__
    self.createTabs()
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/dialog.py", line 124, in createTabs
    self.font_command_tab = fontcommand.FontCommandWidget(self)
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/fontcommand.py", line 189, in __init__
    raise Exception("Unsupported widget")
Exception: Unsupported widget
Error in sys.excepthook:
Traceback (most recent call last):
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/app.py", line 224, in excepthook
    exception.ExceptionDialog(exctype, excvalue, exctb)
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/exception.py", line 67, in __init__
    app.translateUI(self)
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/app.py", line 165, in translateUI
    obj.translateUI()
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/exception.py", line 71, in translateUI
    extension = self._ext_maintainer[0]
TypeError: 'NoneType' object is not subscriptable

Original exception was:
Traceback (most recent call last):
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/__init__.py", line 67, in showAvailableFonts
    dialog.show_fonts_dialog(self.mainwindow())
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/dialog.py", line 58, in show_fonts_dialog
    dlg = FontsDialog(mainwin)
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/dialog.py", line 92, in __init__
    self.createTabs()
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/dialog.py", line 124, in createTabs
    self.font_command_tab = fontcommand.FontCommandWidget(self)
  File "/home/uliska/git/lilypond/frescobaldi/frescobaldi_app/fonts/fontcommand.py", line 189, in __init__
    raise Exception("Unsupported widget")
Exception: Unsupported widget
```

The first and the last part (about the "Unsupported widget") are irrelevant for this issue, they show the triggering cause, in this case a deliberately triggered exception. The problem is the `Error in sys.excepthook:` where the notification/report dialog should be created instead. The text of the message makes me suspect that I've broken it with the Extension API (it looks like it tries to retrieve the extension's maintainer although the exception isn't from an extension in the first place).

I'll look into that ASAP.

**Comment by uliska** [comment]
Fixed in 015b6e49bbad0105a617719186296b7530b4334e

**Comment by wbsoft** [comment]
it was not completely fixed, but I fixed it in 6951e52bcae1f9c50a5049fa3c039d1a370254d6 :-)

The problem was that self.infos(extension) could return None, but the method self.is_extension_exception() didn't count on that.
